src/main/java/com/naveenautomation/Pages/MyAccountPage.java

```diff
@@ -7,7 +7,7 @@
 
 public class MyAccountPage extends Page {
 
-	private static final String PAGE_URL="/account";
+	private static final String PAGE_URL="account/account";
```

Comment:
if account is common in all URL, it should be in base URL and not in page url

Reply:
URL changed as recommended.